soc/arm64/bcm2712/Kconfig.defconfig

```diff
@@ -8,7 +8,7 @@ config SOC
 
 config NUM_IRQS
 	int
-	default 280
+	default 480
```

Comment:
looks like you take into account only SPI IRQs, what about PPIs?

Reply:
Fixed